### PR TITLE
improve ridp status badges

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: 
https://www.pivotaltracker.com/story/show/187638343 && 
https://www.pivotaltracker.com/story/show/187638334 
https://www.pivotaltracker.com/story/show/187638326
https://www.pivotaltracker.com/story/show/187638331

# A brief description of the changes

Current behavior: ridp status badges are not matching the mocked versions on Figma (https://www.figma.com/design/Z1WtgrjH3zB9Y7HQVn0hL5/EA-nuharbor-1?node-id=2462-6012&t=IBqd6tAARVF3FRaD-0), still using outdated styling

New behavior: ridp badges have been improved to match Figma

Screenshot 2024-05-22 at 5 54 58 PM
Screenshot 2024-05-22 at 5 59 40 PM
Screenshot 2024-05-22 at 6 01 08 PM
Screenshot 2024-05-22 at 6 23 13 PM

_if we wanted to improve the mobile view of the button, we could add_ `width: fit-content !important`. _There are probably other solutions but I wasn't sure how important this change is_

### CODE REVIEW

**Comment by ATBull81** [comment]
Quickly confirming whether we want badges to match alert color scheme with Alex Rekas, do not merge or review until then

**Comment by ATBull81** [comment]
> Quickly confirming whether we want badges to match alert color scheme with Alex Rekas, do not merge or review until then

Alex confirmed that the ridp statuses should be this color scheme, PR is ready for review

**Review by kristinmerbach — APPROVED** [review]
LGTM!

File: app/javascript/css/colors.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@
   --success-tint: hsl(130, 61%, 86%);
   --warning-color: hsl(44, 100%, 45%);
   --warning-tint: hsla(51, 100%, 94%, 1);
+  --danger-shade: hsl(0, 100%, 91%);
```
Comment (charlienparker) [inline app/javascript/css/colors.scss:26]: naming nitpick - these are more colors than shades, right? at least I think "shade" in the context of the other status colors we use for banners is used to color the text. Probably makes more sense to call these "colors". And to emphasize the difference that these are similar but different than our other status colors, calling them something like `danger-color-2" etc. is probably better IMO

Comment (ATBull81) [reply app/javascript/css/colors.scss]: https://github.com/ideacrew/enroll/pull/3884/commits/ce3271eb9ccf51b4f4b19769b1b2518291e1ba14

**Review by kristinmerbach — APPROVED** [review]

**Review by charlienparker — APPROVED** [review]

**Pull request merged**